### DISCUSSION TITLE
Is it available to get efficiency for DRS modification?

### DISCUSSION THREAD

**Discussion by UlyanaKa** [NibuTake/PyDEA#25] [Q&A] [answered]
Hello! I want to get efficiency meanings for DRS modification DEA but I didn't find any information in tutorial how to do that. 
Is it available to do it by this instrument? And if yes, how to do it?

**Comment by NibuTake** [comment]
@UlyanaKa 
Hello! Thank you for your question.
Unfortunately, we currently only support CRS and VRS, so it is not available to get efficiency with DRS at this time. But, implementing this feature is not difficult, and we will add it within a few days. Please bear with us for a little while.

**Reply by UlyanaKa** [reply]
Thank you for your answer! I'll be looking forward to this addition)

**Reply by NibuTake** [reply, marked as answer]
@UlyanaKa 
We have just added support for DRS and IRS as RTS (frontier) options. This feature is available in the latest version (1.0.2). We welcome any further feedback or requests you may have! 

https://github.com/NibuTake/PyDEA/releases/tag/release1.0.2

**Reply by UlyanaKa** [reply]
Thank you very much for your brilliant work. For me the speed of response was amazing.
